components/CollectiveNavbar.js

```diff
@@ -230,6 +264,22 @@ const getCollectiveTypeBlacklistedSections = collectiveType => {
  * @param {boolean} `isAdmin` wether the user is an admin of the collective
  */
 export const getSectionsForCollective = (collective, isAdmin) => {
+  // static sections for Events until new settings page for events are implemented
+  if (collective.type === CollectiveType.EVENT) {
+    const EventSections = [
+      Sections.ABOUT,
+      Sections.TICKETS,
+      Sections.CONTRIBUTE,
+      Sections.PARTICIPANTS,
+      Sections.LOCATION,
+      Sections.BUDGET,
+    ];
+
+    if (canOrderTicketsFromEvent(collective)) return EventSections;
+
+    return EventSections.filter(section => section !== Sections.TICKETS);
+  }
```

Comment:
Settings page won't allow to edit sections for the event, at least not for the first iteration (there no UI for it neither for Collective atm). I'd prefer if we can rely on the same blacklist system that we use for collectives, users and orgs to do this - even if that means passing the full `collective` object to `getCollectiveTypeBlacklistedSections`.

Reply:
Makes sense! Pushed a fix to use the blacklist system properly. 

This introduced the issue of getting a good order of the sections for events, without messing up the order for other types of collectives. I'm making the assumption that we want the `About` section at the top of the page, would you agree with that?

I added a `if (collective.type === CollectiveType.EVENT)` that sorts the `About` section to the top, but maybe there is a point of making a more general approach to introduce custom order/sorting for different collective types? The other sections I was able to move in the constants list as they did not affect any other type of collective.

Reply:
> I'm making the assumption that we want the About section at the top of the page, would you agree with that?

Sounds reasonable!

> I added a if (collective.type === CollectiveType.EVENT) that sorts the About section to the top, but maybe there is a point of making a more general approach to introduce custom order/sorting for different collective types?

That should be good enough for now